openmoc/__init__.py

```diff
@@ -32,6 +32,7 @@
 time = (now.month, now.day, now.year, now.hour, now.minute, now.second)
 year_string = '-'.join(map(left_pad, map(str, (now.month, now.day, now.year))))
 today_string = ':'.join(map(left_pad, map(str, (now.hour, now.minute, now.second))))
+today_string += ':' + str(now.microsecond)[:2]
```

Comment:
Have you run into cases where this is needed to avoid having multiple runs write to the same log file? If so, do you think there should be some kind of check to see whether a log file with the desired name already exists?